src/main/java/org/prebid/server/spring/config/bidder/AdformConfiguration.java

```diff
@@ -37,6 +39,9 @@
     @Value("${external-url}")
     private String externalUrl;
 
+    @Value("${adapters.adform.deprecated-names:}")
```

Comment:
Let's stick with approach accepted throughout other configurations and avoid default values in `@Value` annotations, better put it in application.yaml packaged in JAR file.

This applies to all other bidders as well.

Reply:
Yes, but in this case we don't have default value for property.

Reply:
Empty list should be default and it could be specified in configuration file

Reply:
Yes, it can be like:
```
adapters:
  rubicon:
    ...
    deprecated-names:
```
So, set just no value.